lib/project/cordova-project.ts

```diff
@@ -73,7 +73,7 @@ export class CordovaProject extends FrameworkProjectBase implements Project.IFra
 	}
 
 	public get startPackageActivity(): string {
-		return StartPackageActivityNames.CORDOVA;
+		return startPackageActivityNames["cordova"];
```

Comment:
Since we have `$projectConstants` can we use them here: `return startPackageActivityNames[this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova.toLowerCase()];`?